.github/workflows/ci-python.yml

```diff
@@ -43,27 +41,26 @@ jobs:
         run: |
           poetry run python -m unittest -v
         env:
-          NASLINTER_DOCKER_RUN: "true"
-
+          NASLINTER_DOCKER_RUN: 'true'
   codecov:
     name: Upload coverage to codecov.io
     needs: test
-    runs-on: 'ubuntu-latest'
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Install python, poetry and dependencies
         uses: greenbone/actions/poetry@v3
         with:
-          python-version: 3.9
+          python-version: 3.10
```

Comment:
```suggestion
          python-version: '3.10'
```
To fix Codecov

Reply:
done in 289b4a756c87fa99cf59d843f8c80a42062e7ec2